tools/ci/tc/decision.py

```diff
@@ -101,14 +100,22 @@ def get_extra_jobs(event):
     return jobs
 
 
-def filter_jobs(jobs, event):
-
-    # Exclude browser stability tests for exports from that specific browser.
+def filter_excluded_users(tasks, event):
+    # Some users' pull requests are excluded from tasks,
+    # such as pull requests from automated exports.
     try:
-        if event["pull_request"]["user"]["login"] == "chromium-wpt-export-bot":
-            jobs.discard("wpt-chrome-dev-stability")
-        if event["pull_request"]["user"]["login"] == "moz-wptsync-bot":
-            jobs.discard("wpt-firefox-nightly-stability")
+        submitter = event["pull_request"]["user"]["login"]
+        excluded_tasks = []
+        for name, task in list(tasks.items()):
```

Comment:
Is the `list()` wrapping here necessary?

Reply:
Because we're removing items from the dictionary `tasks` during iteration, we have to have a separate reference to iterate or we will raise a `RuntimeError: dictionary changed size during iteration.`


https://github.com/web-platform-tests/wpt/blob/4b855de437c7f4bf1aab26dfe4cc46eccb023004/tools/ci/tc/decision.py#L109-L113

Reply:
Aha, no mistake then! A comment would be good though, since it's not very obvious if the `list()` is because `tasks.items()` isn't a thing that can be directly iterated (its type isn't visible here) or because `tasks` is mutated.

Reply:
I agree, as I had to find a way to handle this myself when implementing - comment added